sdk/tests/js/tests/drawingbuffer-storage-test.js

```diff
@@ -0,0 +1,275 @@
+/*
+Copyright (c) 2023 The Khronos Group Inc.
+Use of this source code is governed by an MIT-style license that can be
+found in the LICENSE.txt file.
+*/
+
+"use strict";
+
+let gl;
+let oldViewport;
+let width;
+let height;
+let format;
+let hasDrawingBufferStorage;
+let maxRenderbufferSize;
+
+function runTest(contextVersion) {
```

Comment:
This `contextVersion` parameter is unused, which prevents runTest(1) and runTest(2) from testing both WebGL 1 and 2 contexts. I filed #3719 for this and another issue related to RGBA16F that I spotted.